functional/config.rb

```diff
@@ -0,0 +1,58 @@
+# Size of the CoreOS cluster created by Vagrant
```

Comment:
Well. Let's assume we will add libvirt support for the original Vagrantfile. If we make all modifications (channel, memory, additional cases) it will be harder to merge new changes into this Vagrantfile.

Additional `config.vm.provision` string is an exception. Because without it we have to add more logic into our scripts (i.e. create lock file which will tell `sudo ./test` script that go binaries are downloading and wait until this lock file will be removed, then continue with the tests). If you are ok with these changes - I will do that.